### DISCUSSION TITLE
assetlinks.json file on root

### DISCUSSION THREAD

**Discussion by aureabelle** [vercel/next.js#13113] [Help] [answered]
Hi everyone! I really need help on having a json file accessible on the root of my app, without having it return a 404. 

Basically, there's a requirement for our Android app to have a json file accessible in the website.  Example: https://mywebsite.com/.well-known/assetlinks.json.

"/.well-known/" is the directory that sits on the root folder and contains the "assetlinks.json" file. 

Any help is much appreciated. Thank you!

**Comment by rafaelalmeidatk** [comment, marked as answer]
You can add this folder inside the `public` folder, this will make it available directly from the root of your app: https://nextjs.org/docs/basic-features/static-file-serving

**Reply by aureabelle** [reply]
Thanks @rafaelalmeidatk for the quick response. I'll give this a try.

**Reply by aureabelle** [reply]
Works like a charm. Thanks!

**Reply by nicolaspda** [reply]
Omg, i was stuck for 2 days, trying to do this but with Nuxt.js.
It turns out this awnser lead me to the right track. The "Public" folder in nuxt is "Static".

Thanks!

**Comment by TheyCallMeFelix** [comment]
Also make sure you have file named .well-known and not .wellknown sounds dumb but it happens all the time
